src/test/resources/expected-classes/CommitResolver.java.txt

```diff
@@ -7,14 +7,19 @@ public interface CommitResolver {
 
     PullRequestConnection associatedPullRequests(Commit commit, String after, String before, Integer first, Integer last, PullRequestOrder orderBy, DataFetchingEnvironment env) throws Exception;
 
+    @javax.validation.constraints.NotNull
+    @com.fasterxml.jackson.databind.annotation.JsonDeserialize(using = com.example.json.DateTimeScalarDeserializer.class)
```

Comment:
@joffrey-bion I think its a bit late to ask but why do we have this annotation here?